spec/takeaway_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'takeaway'
+require 'dotenv'
+
+describe Takeaway do
+  subject(:takeaway) { described_class.new(menu, calculator, twilio, order) }
+  let(:order) { double :order}
+  let(:calculator) { double :calculator }
+  let(:choice) { {beef:2, rolls:3} }
+  let(:twilio) { double :twilio }
+  let(:confirm) { "order confirmed: correct amount billed 17" }
+  let(:menu) { double :menu, show: {ribs: 3,
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.